apps/organizations/models.py

```diff
@@ -320,18 +320,47 @@ def status(self) -> str:
     ##
     # Notifications
 
+    def _build_notification_message(self, subject: User, verb: str) -> str:
+        """Builds a message that will be displayed as an internal Slack notification.
+        """
+        msg = '{subject_name} ({subject_username}<{subject_profile_url}|{email}>) has {verb} an invitation for Organization <{organization_url}|{organization_name}>'.format( # noqa: E501
+            verb=verb,
+            subject_name=subject.profile.get_full_name(),
+            subject_username=subject.username,
+            subject_profile_url=subject.profile.profile_url,
+            email=subject.email,
+            organization_url=self.organization.profile_url,
```

Comment:
Remove this, because there is no such attribute on this class. It _might_ exist on a descendant, but it is not on this (ancestor) class -- that's where you might be getting confused.